playbooks/group_vars/all/global.yml

```diff
@@ -1,108 +1,37 @@
 ---
 # run roles based on certain params
+skip_clean: false
+skip_packages: false
 skip_prereqs: false
-force_minishift_install: false
-setup_minishift: true
-start_minishift: true
-force_repo_clone: true
-setup_jenkins: true
-setup_fedmsg: true
-setup_containers: true
+skip_artifacts: false
+skip_minishift: false
 
-# default location for minishift
-minishift_dest_dir: "{{ ansible_env.HOME }}/minishift"
-
-# default minishift profile name
-profile: minishift
-
-# disk and memory size
-disk_size: 40gb
-memory: 6400mb
-
-# base device size
-# for rootfs in pods
-basedevice_size: 20G
-
-# minishift iso location
-minishift_iso: http://artifacts.ci.centos.org/fedora-atomic/minishift/iso/minishift.iso
-
-# repo to clone
-pipeline_repo: https://github.com/CentOS-PaaS-SIG/ci-pipeline
-
-# repo directory
-pipeline_dir: "{{ ansible_env.HOME }}/minishift/ci-pipeline"
-
-# repo refspec
-pipeline_refspec: '+refs/pull/*:refs/heads/*'
-
-# repo branch or sha
-pipeline_branch: 'master'
+# libvirt group used in roles/prereqs/tasks/install_kvm_plugin.yml
+remote_user: "{{ ansible_user | default(lookup('env', 'USER')) }}"
 
-# cluster username
-username: developer
+# kvm plugin url
+kvm_plugin_url: "https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.10.0/docker-machine-driver-kvm-centos7"
 
-# cluster password
-password: developer
+# oc url
+oc_release: "v3.7.1"
+oc_url: "https://firemanxbr.fedorapeople.org/contra-env-setup/openshift-origin-client-tools-{{ oc_release }}-ab0f056-linux-64bit.tar.gz"
 
-# cluster admin username
-admin_username: system
+# default location for oc
+oc_dest_dir: "{{ ansible_env.HOME }}/.minishift/cache/oc/v3.7.1/linux"
```

Comment:
What if someone doesn't install minishift?

Reply:
The idea of contra-env-setup is offering a complete local environment to the users if the user uses: 'skip_minishift=true' will not install it. This complement doesn't affect anything before or after but offer the vision we don't have today.
This add-ons offer the admin access to minishift permitting the engineer look and debug including the 'kube-system' and 'default'. Can be useful in some cases but If do you would like removed it, please let me know.